### PR TITLE
Upgrades: Ruby 2.4.1, Rails point release, Nokogiri point release

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

Futzing around with a ruby upgrade to 2.4.0 and a rails upgrade to 4.2.8, plus a `rake rails:update`. Stems from #853

### CODE REVIEW

**Comment by colinxfleming** [comment]
Looks like factory_girl is causing a segfault. yikes.

**Comment by colinxfleming** [comment]
```
Colins-Macbook-Pro:dcaf_case_management colin$ rake test
Started with run options --seed 56501

/Users/colin/.rbenv/versions/2.4.0/lib/ruby/2.4.0/forwardable.rb:228: [BUG] Segmentation fault at 0x00000000000038                                                              ] 7% Time: 00:00:29,  ETA: 00:05:58
ruby 2.4.0p0 (2016-12-24 revision 57164) [x86_64-darwin15]

-- Crash Report log information --------------------------------------------
   See Crash Report log file under the one of following:                    
     * ~/Library/Logs/CrashReporter                                         
     * /Library/Logs/CrashReporter                                          
     * ~/Library/Logs/DiagnosticReports                                     
     * /Library/Logs/DiagnosticReports                                      
   for more details.                                                        
Don't forget to include the above Crash Report log file in bug reports.     

-- Control frame information -----------------------------------------------
c:0117 p:0045 s:0632 e:000629 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/2.4.0/forwardable.rb:228
c:0116 p:0022 s:0623 E:0001d8 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/server/connection.rb:194 [FINISH]
c:0115 p:---- s:0619 e:000618 CFUNC  :each
c:0114 p:0029 s:0615 E:0001a8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/server/connection.rb:192
c:0113 p:0010 s:0608 E:000268 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/server/connection.rb:162
c:0112 p:0010 s:0603 E:0002f8 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/server/connection.rb:108
c:0111 p:0101 s:0599 E:0003d0 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/monitoring/publishable.rb:47
c:0110 p:0050 s:0585 E:0002d0 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/server/connection.rb:107
c:0109 p:0055 s:0579 E:000470 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/operation/executable.rb:37
c:0108 p:0016 s:0573 E:0004c8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/server/connection_pool.rb:107
c:0107 p:0014 s:0568 E:000558 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/server.rb:242
c:0106 p:0010 s:0563 E:001588 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/operation/executable.rb:35
c:0105 p:0028 s:0558 E:0005e8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/collection/view/iterable.rb:81
c:0104 p:0027 s:0553 E:000650 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/collection/view/iterable.rb:41 [FINISH]
c:0103 p:0014 s:0548 E:0006e8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/retryable.rb:51
c:0102 p:0015 s:0541 E:000f10 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongo-2.4.1/lib/mongo/collection/view/iterable.rb:39
c:0101 p:0033 s:0537 E:0007b0 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/query_cache.rb:207 [FINISH]
c:0100 p:---- s:0531 E:0006f0 CFUNC  :first
c:0099 p:0019 s:0527 E:000810 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/contextual/mongo.rb:148
c:0098 p:0057 s:0524 E:0008a8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/contextual/mongo.rb:488
c:0097 p:0062 s:0517 E:001f30 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/contextual/mongo.rb:147
c:0096 p:0022 s:0513 E:000950 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/contextual.rb:20
c:0095 p:0124 s:0505 E:0009c8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/validatable/uniqueness.rb:277
c:0094 p:0072 s:0496 E:000a70 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/validatable/uniqueness.rb:48
c:0093 p:0040 s:0491 E:000b00 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/validatable/queryable.rb:23
c:0092 p:0011 s:0485 E:000a38 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/validatable/uniqueness.rb:42
c:0091 p:0068 s:0478 E:000bd0 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activemodel-4.2.8/lib/active_model/validator.rb:151 [FINISH]
c:0090 p:---- s:0473 e:000472 CFUNC  :each
c:0089 p:0012 s:0469 E:000b98 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activemodel-4.2.8/lib/active_model/validator.rb:148 [FINISH]
c:0088 p:---- s:0464 e:000463 CFUNC  :public_send
c:0087 p:0016 s:0458 E:000c68 LAMBDA /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:455 [FINISH]
c:0086 p:0041 s:0452 E:000cd8 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:182 [FINISH]
c:0085 p:0013 s:0446 E:000d68 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:504 [FINISH]
c:0084 p:---- s:0442 e:000441 CFUNC  :each
c:0083 p:0011 s:0438 E:000d40 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:504
c:0082 p:0066 s:0432 E:000e10 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:92
c:0081 p:0015 s:0424 E:000e68 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:778
c:0080 p:0008 s:0419 E:000ec0 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activemodel-4.2.8/lib/active_model/validations.rb:399
c:0079 p:0010 s:0415 E:000f10 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activemodel-4.2.8/lib/active_model/validations/callbacks.rb:113 [FINISH]
c:0078 p:0041 s:0412 E:000f70 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:117
c:0077 p:0018 s:0405 E:000fc8 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:555 [FINISH]
c:0076 p:0024 s:0401 E:001030 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:505
c:0075 p:0066 s:0395 E:0010a0 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:92
c:0074 p:0015 s:0387 E:001138 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:778
c:0073 p:0009 s:0382 E:002570 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activemodel-4.2.8/lib/active_model/validations/callbacks.rb:113
c:0072 p:0040 s:0378 E:0011d0 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activemodel-4.2.8/lib/active_model/validations.rb:338
c:0071 p:0033 s:0372 E:001228 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/validatable.rb:97
c:0070 p:0013 s:0367 E:001288 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activemodel-4.2.8/lib/active_model/validations.rb:375
c:0069 p:0033 s:0362 E:001350 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/persistable/creatable.rb:115
c:0068 p:0015 s:0356 E:0012e8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/persistable/creatable.rb:23
c:0067 p:0022 s:0351 E:0013e8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/persistable/savable.rb:23
c:0066 p:0014 s:0346 E:001448 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/mongoid-5.0.2/lib/mongoid/persistable/savable.rb:44
c:0065 p:0009 s:0341 E:0014a8 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/configuration.rb:18 [FINISH]
c:0064 p:0012 s:0337 E:001508 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/evaluation.rb:15
c:0063 p:0035 s:0332 E:001598 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/strategy/create.rb:12 [FINISH]
c:0062 p:---- s:0328 e:000327 CFUNC  :tap
c:0061 p:0013 s:0324 E:001568 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/strategy/create.rb:9
c:0060 p:0145 s:0319 E:002398 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/factory.rb:42
c:0059 p:0017 s:0308 E:001690 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/factory_runner.rb:29
c:0058 p:0047 s:0305 E:001720 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/notifications.rb:166
c:0057 p:0102 s:0299 E:001670 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/factory_runner.rb:28
c:0056 p:0026 s:0291 E:0017c8 LAMBDA /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/strategy_syntax_method_registra [FINISH]
c:0055 p:0018 s:0285 E:001820 BLOCK  /Users/colin/repos/dcaf_case_management/test/factories/patient.rb:7 [FINISH]
c:0054 p:---- s:0282 e:000281 CFUNC  :instance_exec
c:0053 p:0036 s:0278 E:001878 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/attribute/dynamic.rb:16 [FINISH]
c:0052 p:---- s:0274 e:000273 CFUNC  :instance_exec
c:0051 p:0041 s:0270 E:0018d0 LAMBDA /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/evaluator.rb:77 [FINISH]
c:0050 p:0012 s:0264 E:001928 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/attribute_assigner.rb:56
c:0049 p:0019 s:0259 E:0019d8 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/attribute_assigner.rb:16 [FINISH]
c:0048 p:---- s:0253 e:000252 CFUNC  :each
c:0047 p:0012 s:0249 E:0019a8 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/attribute_assigner.rb:15 [FINISH]
c:0046 p:---- s:0245 e:000244 CFUNC  :tap
c:0045 p:0025 s:0241 E:001980 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/attribute_assigner.rb:14
c:0044 p:0021 s:0237 E:001ab0 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/evaluation.rb:12
c:0043 p:0009 s:0229 E:001b08 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/strategy/create.rb:9
c:0042 p:0145 s:0224 E:002208 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/factory.rb:42
c:0041 p:0017 s:0213 E:001bd0 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/factory_runner.rb:29
c:0040 p:0047 s:0210 E:001c60 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/notifications.rb:166
c:0039 p:0102 s:0204 E:001bb0 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/factory_runner.rb:28
c:0038 p:0026 s:0196 E:001d08 LAMBDA /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/factory_girl-4.8.0/lib/factory_girl/strategy_syntax_method_registra [FINISH]
c:0037 p:0046 s:0190 E:001230 BLOCK  /Users/colin/repos/dcaf_case_management/test/models/patient_test.rb:9 [FINISH]
c:0036 p:---- s:0187 e:000186 CFUNC  :instance_eval
c:0035 p:0011 s:0183 E:001280 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-spec-rails-5.4.0/lib/minitest-spec-rails/dsl.rb:19 [FINISH]
c:0034 p:---- s:0180 e:000179 CFUNC  :instance_exec
c:0033 p:0012 s:0176 E:0012e0 LAMBDA /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:446 [FINISH]
c:0032 p:0019 s:0171 E:001338 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:192 [FINISH]
c:0031 p:0013 s:0167 E:0013c8 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:504 [FINISH]
c:0030 p:---- s:0163 e:000162 CFUNC  :each
c:0029 p:0011 s:0159 E:0013a0 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:504
c:0028 p:0066 s:0153 E:001470 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:92
c:0027 p:0015 s:0145 E:0014c8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:778
c:0026 p:0020 s:0140 E:001530 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:81
c:0025 p:0019 s:0134 E:001580 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/activesupport-4.2.8/lib/active_support/testing/setup_and_teardown.r
c:0024 p:0010 s:0130 E:0015d0 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest/spec.rb:312
c:0023 p:0008 s:0126 E:001660 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest/test.rb:103
c:0022 p:0006 s:0123 E:0016e8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest/test.rb:202
c:0021 p:0009 s:0118 E:001640 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest/test.rb:102
c:0020 p:0020 s:0115 E:0017a8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest/test.rb:253
c:0019 p:0009 s:0110 E:001620 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest/test.rb:101
c:0018 p:0037 s:0107 E:0019f0 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest.rb:349
c:0017 p:0052 s:0099 E:0024e8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest/test.rb:273
c:0016 p:0009 s:0092 E:001640 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest/test.rb:100
c:0015 p:0063 s:0088 E:0018d8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-reporters-1.1.14/lib/minitest/reporters.rb:48
c:0014 p:0014 s:0081 E:001948 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest.rb:822
c:0013 p:0032 s:0074 E:0019b8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest.rb:323
c:0012 p:0014 s:0066 E:001a18 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest.rb:310 [FINISH]
c:0011 p:---- s:0062 e:000061 CFUNC  :each
c:0010 p:0010 s:0058 E:001c70 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest.rb:309
c:0009 p:0037 s:0055 E:0020b0 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest.rb:349
c:0008 p:0035 s:0047 E:0008d8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest.rb:336
c:0007 p:0131 s:0040 E:000898 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest.rb:308
c:0006 p:0013 s:0031 E:001868 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest.rb:158 [FINISH]
c:0005 p:---- s:0027 e:000026 CFUNC  :map
c:0004 p:0046 s:0023 E:000658 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest.rb:158
c:0003 p:0158 s:0014 E:0022f8 METHOD /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest.rb:135
c:0002 p:0071 s:0007 E:001f98 BLOCK  /Users/colin/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/minitest-5.10.1/lib/minitest.rb:62 [FINISH]
c:0001 p:0000 s:0003 E:001070 (none) [FINISH]
```

**Comment by colinxfleming** [comment]
Segfault is resolved with 2.4.1 (thanks @DarthHater ). A nokogiri upgrade was failing the build but that's fixed in 4c53d5d . Now resolving a failing integration test on call list.

**Comment by DarthHater** [comment]
+1, looks good. Wonder what it was in 2.4.1 that fixed the Segfault.

File: config/environments/test.rb [hunk 1]
```diff
@@ -13,7 +13,8 @@
   config.eager_load = false
 
   # Configure static file server for tests with Cache-Control for performance.
-  config.serve_static_files   = false
+  # Changed to true by rake rails:update
+  config.serve_static_files   = true
```
Comment (DarthHater) [inline config/environments/test.rb]: This is the only thing that catches me off guard. What was serving them before?

Comment (colinxfleming) [inline config/environments/test.rb]: That's a good question! I THINK what this does is makes stuff in the public folder accessible, but I think it'll work fine for us either way.

Comment (colinxfleming) [inline config/environments/test.rb]: c.f. https://stackoverflow.com/questions/17550747/rails-4-enabled-rails-to-serve-static-assets-is-it-correct-on-heroku

**Comment by colinxfleming** [comment]
All clear! Merging.

**Pull request merged**